DOL.WHD.Section14c.Web/src/modules/pages/landingPageTemplate.html

```diff
@@ -1,27 +1,18 @@
 <dol-header></dol-header>
 
 <main tabindex="-1" id="mainContent" role="main" class="main-content">
-  <div ng-if="!stateService.user.email">
-    <div class="dol-landing-public">
-      <h1>Section 14(c) Certificate Application</h1>
-
-      <div class="dol-width-one-half">
-        <user-login></user-login>
-      </div>
-      <div class="dol-width-one-half">
-        <user-registration-form></user-registration-form>
-      </div>
-    </div>
-  </div>
-
-  <div ng-if="stateService.hasClaim('Application.Submit')">
+ 
+  <div>
     <main-top-nav-control></main-top-nav-control>
-    <div ng-show="!stateService.applicationStarted()">
-      <button class="save-button" ng-click="navToApplication()">Start A New Application</button>
-    </div>
-    <div ng-show="stateService.applicationStarted()">
-      <button class="save-button" ng-click="navToApplication()">Continue Application</button>
+    <div ng-repeat="organization in organizations track by $index">
+      <div >
+        <button class="save-button" ng-click="onApplicationClick($index)">
+          <span ng-show="organization.applicationStatus.name === 'New'">Start A New</span> 
+          <span ng-show="organization.applicationStatus.name === 'InProgress'">Continue</span> 
+          <span ng-show="organization.applicationStatus.name === 'Submitted'">View</span> Application for {{organization.employer.legalName}}</button>
```

Comment:
`</button>` should be on the next line. (I'm picky picky, I know.)

Also, for the `New` and `InProgress` texts, should they include the organization's name somewhere so the user will know which organization they're looking at instead of just `Start A New`?  (The `Submitted` state includes the organization name in the description.)